src/Codeception/Module/REST.php

```diff
@@ -575,11 +575,13 @@ public function sendUNLINK($url, array $linkEntries)
     protected function execute($method, $url, $parameters = [], $files = [])
     {
         // allow full url to be requested
-        if (strpos($url, '://') === false) {
-            $url = $this->config['url'] . $url;
-            if ($this->config['url'] && strpos($url, '://') === false && strpos($url, '/') !== 0 && $this->config['url'][0] !== '/') {
-                $url = '/' . $url;
-            }
+        if (strpos($url, '://') === false && $this->config['url']) {
+            $url = $this->config['url']
+             . (substr($this->config['url'], -1) != '/' ? '/' : '')
+             . substr($url, substr($url, 0, 1) == '/' ? 1 : 0);
+        }
+        if (strpos($url, '://') === false && strpos($url, '/') !== 0) {
```

Comment:
Is this line ever reachable?

AFAIK: `$this->config['url']` always has value. 
So `strpos($url, '://') === false && $this->config['url']` should be true for every `strpos($url, '://') === false`, and it will change value of $url so this block won't be executed.

Anyway, could you add tests for REST module so we could be sure about all possible cases handled by this change>

Reply:
I don't know why it was there, I had high level of temptation to just delete it. But thought there was a reason for it and it was not making harm.
The origin of that line is #4796

Reply:
Anyway, could you write a test cases for this in `tests/unit/Codeception/Module/RestTest.php`?

Reply:
done